app/views/shelters/index.html.erb

```diff
@@ -4,7 +4,9 @@
   <section id= 'shelter-<%= shelter.id %>'>
     <p>-----<%= link_to "#{shelter.name}", "/shelters/#{shelter.id}" %>-----</p>
     <%= link_to "Edit #{shelter.name}", "shelters/#{shelter.id}/edit" %> /
-    <%= link_to "Delete #{shelter.name}", "shelters/#{shelter.id}", method: :delete %>
+    <% if !shelter.approved_pet? %>
```

Comment:
checks if the `shelter` has pets that have been approved and adjust the view based on that